frontend/public/components/network-policies/create-network-policy.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import { getActiveNamespace } from '../../actions/ui';
+import { NetworkPolicy } from './network-policy-model';
+import { NetworkPolicyForm } from './network-policy-form';
+
+export const CreateNetworkPolicy: React.FunctionComponent<{}> = () => {
+  const { t } = useTranslation();
+  const emptyPolicy: NetworkPolicy = {
+    name: '',
+    namespace: getActiveNamespace(),
+    podSelector: [['', '']],
+    ingress: {
+      denyAll: false,
+      rules: [],
+    },
+    egress: {
+      denyAll: false,
+      rules: [],
+    },
+  };
+  const [networkPolicy, setNetworkPolicy] = React.useState(emptyPolicy);
```

Comment:
That's correct (I was just a little bit ahead of time because in a future PR, this component will have more to do with networkPolicy ; but you're totally right if we consider strictly this commit)
I'll change that